src/envoy/http/path_matcher/filter.cc

```diff
@@ -94,19 +93,14 @@ Envoy::Http::FilterHeadersStatus Filter::decodeHeaders(
   utils::setStringFilterState(filter_state, utils::kFilterStateOperation,
                               operation);
 
-  if (rule->has_path_parameter_extraction()) {
-    const PathParameterExtractionRule& param_rule =
-        rule->path_parameter_extraction();
+  std::vector<VariableBinding> variable_bindings;
+  config_->findRule(method, path, &variable_bindings);
 
-    std::vector<VariableBinding> variable_bindings;
-    config_->findRule(method, path, &variable_bindings);
-
-    if (!variable_bindings.empty()) {
-      const std::string query_params = VariableBindingsToQueryParameters(
-          variable_bindings, param_rule.snake_to_json_segments());
-      utils::setStringFilterState(filter_state, utils::kFilterStateQueryParams,
-                                  query_params);
-    }
+  if (!variable_bindings.empty()) {
```

Comment:
This is not correct.  We only do path translation for CONSTANT case.

Reply:
ACK